packages/block-editor/src/hooks/font-size.js

```diff
@@ -153,6 +154,54 @@ export function useIsFontSizeDisabled( { name: blockName } = {} ) {
 	);
 }
 
+/**
+ * Add inline styles for font sizes.
+ * Ideally, this is not needed and themes load the font-size classes on the editor.
+ *
+ * @param  {Function} BlockListBlock Original component
+ * @return {Function}                Wrapped component
+ */
+const withFontSizeInlineStyles = createHigherOrderComponent(
+	( BlockListBlock ) => ( props ) => {
+		const {
+			name: blockName,
+			attributes: { fontSize, style },
+			wrapperProps,
+		} = props;
+
+		const { fontSizes } = useSelect( ( select ) =>
```

Comment:
@ellatrix should we do the same you did for the colors here?

Reply:
Yes, sorry, was relocating just now.